### ISSUE TITLE
Phase 2 Block 1 compatibility - 4th Pass

### ISSUE THREAD

**Issue by mrmin123** [perryhuynh/kcauto#232]
Port Ship Switcher module and deal with any other bugs and performance improvements.

✅  Ship Switcher
✅ Fix panel assets for non-EO maps that have boss bars
✅ Add 7-1 to webUI
✅ Update assets for proper handling of fleets busy with expeditions
✅ Update assets for better fleet resupplying

**Comment by n0k0m3** [comment]
Feature request: 
- Extend level check from ShipSwitcher to repair and refuel level check.
- Option for proceed with taiha'd non-flagship ships (a.k.a. push when taiha just like LastNodePush introduced in the legacy kancolle-auto)

Reason: Orel Cruising is dead since DDs in 2-3 got upgraded and can one hit solo Ro-500. So now the alternative method (courtesy of @Nishisonic on Twitter) is AV with DLC + scrap DDs with drums, sortie to 1-3 and 5-5 until the resource node, then switch the taiha'd scraps with other scraps. 

The only ship gets repaired is the flagship AV (can be rotated when fatigued). Sunking scraps is possible (thus, the 2nd feature request).

Briefing of the method (thanks to Marc Keller on KCUSA facebook group):
```
The Oil Refinery is 1-3 as you all know. And in the HTML 5 version it's changed quite drastically to the point it now has routing mechanics.

The one route that heads up to a fuel area where you can get some fuel there.

But what people may not realize is... You can get additional fuel by equipping extra drums and Daihatsu type equipment on your ships.

Recommended Fleet composition: 1 AV, 5 DD (preferably throwaway so you can make alarming profit)

Equipment: AV (preferable well trained) - Daihatsus (Zuiuns and Hiyoteki optional)
DD - Drum Canisters only

The main thing you do is you go automatically north and should hit the fuel area at the top, Fuel node gives between 10-20 fuel. After that, any additional fuel you get comes from your drums and Daihatsu.

+2 fuel per drum
+3 fuel per Daihatsu.
(Note: All Daihatsu types should work except for Toku Daihatsu + 11 Tank Regiment and Kinu's bonus)

after this your fleet will enter battle and fight against the enemy fleet. Once done, return to base, refuel ONLY your AV, rinse and repeat till your DDs are either critically damaged or run out of fuel and ammo.

If they run out of fuel and ammo or are critically damaged, remove thier drums, scrap, and then bring in another throw away replacement to drum up and run the course.

On average an AV's fuel cost should only be about 8-10 fuel per run (depending on who you run)

With my current set up I had (10 drums + 1 Daihatsu) I was making 33-43 fuel per run.

Each run is roughly about 2 minutes, so if I were to make a full hour run with enough DDs to replace and enough AVs to rotate when they become fatigued. I could easily make roughly 800-1000 fuel per hour with my set up.

Possibly more can be gained if you can utilize Daihatsu properly!

Long story short, the fuel gain when used correctly will dwarf just about any fuel base expedition in a shorter period of time. So you can use this as a means to rapidly gain fuel in over a course of a few days.```

**Comment by mrmin123** [comment]
@minh6a last node push already exists (see `1-6.json`, node D, for the `push` node type specification)

**Comment by mrmin123** [comment]
as reported on Discord, 7-1 has a boss bar
![image](https://user-images.githubusercontent.com/4326866/45070048-5d29eb00-b084-11e8-9ee7-93481dcfc4f4.png)

edit: also, 7-1 is not a valid option on the webui

**Comment by mrmin123** [comment]
@minh6a Having read through the feature request, that isn't something that will be tackled in this ticket. This is a compatibility pass. Please make a separate ticket.
